tests/stubs/talon/__init__.py

```diff
@@ -16,30 +25,78 @@ class Actions:
     """
 
     def __init__(self):
-        self.reset_actions()
+        self.registered_actions = {
+            "module": {"non_namespaced": {}, "namespaced": {}},
```

Comment:
All actions in talon have a namespace -- it's just that there is syntactic sugar for your convenience to refer to actions in the "main" namespace. But you can also refer to them prefixed, e.g. "main.key()". Would be simpler to remove the distinction here and simply use "main" if the namespace is empty. That will let you drop one level of the dictionary.

Reply:
This seems to work for me in .talon land, but not in python (i.e. actions.main.key("t") doesn't work in the repl).